R/utils.R

```diff
@@ -158,10 +170,19 @@ get_selected_cluster_id = function(widgetChoice,
   filename = get_filename(widgetChoice)
   g = readRDS(filename)
   built = suppressWarnings(ggplot2::ggplot_build(g))
-  n_layers = length(built$data)
-  ids = built$data[n_layers][[1]]["data_id"]
-  tooltips = built$data[n_layers][[1]]$tooltip
-  tooltip_ids = suppressWarnings(readr::parse_number(tooltips))
+  g = readRDS(filename)
+  built = suppressWarnings(ggplot2::ggplot_build(g))
```

Comment:
```suggestion
```

Remove the duplicated lines

Reply:
Thanks - I've removed that duplication :)